client/src/components/Settings/SettingsCanvasesView.res

```diff
@@ -21,20 +26,54 @@ let view = (settings: T.t): list<Html.html<'msg>> => {
   }
 
   let canvasView = list{
-    Html.p(list{Attrs.class("canvas-list-title")}, list{Html.text("Personal canvases:")}),
-    Html.div(list{Attrs.class("canvas-list")}, list{canvases}),
-    Html.p(list{}, list{Html.text("Create a new canvas by navigating to the URL")}),
+    Html.p(list{}, list{Html.text("Personal canvases:")}),
+    C.listView(list{canvases}),}
+
+  let createCanvas =  {
+    let field = Html.span(
+      list{},
+      list{
+        C.emailInput(
```

Comment:
This should be an `input`, not an `emailInput`. For LoadableState you can use `LoadStatus.Success("")`